go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/cloudflare/circl
 go 1.12
 
 require (
+	github.com/armfazh/h2c-go-ref v0.0.0-20210215173008-07e12a6f8e0d
```

Comment:
It is better to use the tagged version of h2c.
`go get github.com/armfazh/h2c-go-ref@v10.0.1`

Reply:
versions from v2 and above require a [special path](https://golang.org/ref/mod#minimal-module-compatibility) that matches the version number. 
I confirm that h2c-go-ref points to v10.0.1.